projects/core/src/personalization/services/personalization-context.service.ts

```diff
@@ -0,0 +1,46 @@
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+import { PersonalizationContext } from '../model/personalization-context.model';
+import { filter, map } from 'rxjs/operators';
+import { PersonalizationConfig } from '../config/personalization-config';
+import { CmsService } from '../../cms/facade/cms.service';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class PersonalizationContextService {
+  constructor(
+    private config: PersonalizationConfig,
```

Comment:
please also consider whether customer will extend this service. If yes, please use `protected`.